e2e_test/bin/e2e_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:io';
+
+import 'package:http/http.dart';
+import 'package:sentry/sentry.dart';
+
+const _exampleDsn =
+    'https://8b83cb94764f4701bee40028c2f29e72@o447951.ingest.sentry.io/5428562';
+
+const _org = 'sdk';
+const _projectSlug = 'dart';
```

Comment:
Same question: Which org and which project should be used?

Reply:
the same from the example, org sentry-sdks, project sentry-flutter